docs/readmes/orc8r/dev_minikube.md

```diff
@@ -156,6 +194,27 @@ orc8r-user-grafana-6498bb6959-rchx5              1/1     Running   0          2m
 postgresql-0                                     1/1     Running   4          6d23h
 ```
 
+If using fluentd, there should be a pod `orc8r-fluentd-forward` running as well. Additionally,
+
+```bash
+kubectl --namespace kube-system get pods
+```
+
+should show a pod `orc8r-fluentd-daemon` running.
```

Comment:
Do not split the list of running pods before and after the command. Better give the command to run first and than list all pods you expect.

Reply:
Ok, I added the output with the additional pods created in one block